tests/foreman/cli/test_host.py

```diff
@@ -2711,7 +2711,7 @@ def test_positive_create_host_with_lifecycle_environment_name(
 ):
     """Attempt to create a host with lifecycle-environment name specified
 
-    :id: 7445ad21-538f-4357-8bd1-9676d2478633
+    :id: 8529a272-eeb3-4e92-8032-c0a5e65f8f75
```

Comment:
I have no idea why pre-commit is forcing me to change this ID.

Reply:
Please check if there is a duplicate file in robottelo or in same test exists in some other module.

Reply:
yeah, the reason is that the `UUID` is already used by different test:
`tests/foreman/cli/test_host.py::test_positive_create_host_with_lifecycle_environment_name`

Reply:
Folks, I've already resolved this. The reason was different. What Roman shows is this exact line that the pre-commit wanted to change, and it was because I had a file locally (not in git) that had a test with the same UUID. The script takes into account everything, not just the repo contents.